midcli/command/call_mixin/__init__.py

```diff
@@ -51,6 +51,34 @@ def call(self, name, *args, job=False, raise_=False):
 
             return rv
 
+    def _call_util(self, method, *args, **kwargs):
+        with self.context.get_client() as c:
+            try:
+                return c.call(method, *args, **kwargs)
+            except Exception as e:
+                if (error := self._handle_error(e)) is not None:
+                    raise ProcessInputError(f"Error while calling {method}(*{args!r}, **{kwargs!r}):\n{error}")
+
+                raise
+
+    def _handle_error(self, e):
+        if isinstance(e, ValidationErrors):
+            return ("Validation errors:\n" +
+                    "\n".join([
+                        f"* {error.attribute}: {error.errmsg}" if error.attribute else f"* {error.errmsg}"
+                        for error in e.errors
+                    ]))
+
+        if isinstance(e, ClientException):
+            if e.error.startswith((
+                "[EFAULT] Too many arguments",
+            )) or re.match("\[EFAULT] Command .+ failed \(code", e.error):
```

Comment:
I took the traceback from the ticket and tried locally and I can't get this regex to match.

```
>>> ee = "[EFAULT] Command '('ipmitool', '-c', 'sel', 'elist')'"
>>> re.match('\[EFAULT] Command .+ failed \(code', ee)
>>>

Reply:
Ticket's traceback says
```
subprocess.CalledProcessError: Command '('ipmitool', '-c', 'sel', 'elist')' returned non-zero exit status 1. 
```

Reply:
We are not working with the trace here, we are working with the error itself, and it is different because of https://github.com/truenas/middleware/blob/master/src/middlewared/middlewared/service_exception.py#L86

Reply:
Cool, thanks!